wom/src/main/scala/wdl4s/wdl/expression/TypeEvaluator.scala

```diff
@@ -96,6 +96,8 @@ case class TypeEvaluator(override val lookup: String => WdlType, override val fu
               }
             case ns: WdlNamespace => Success(lookup(ns.importedAs.map{ n => s"$n.${rhs.getSourceString}" }.getOrElse(rhs.getSourceString)))
             case _ => Failure(new WdlExpressionException("Left-hand side of expression must be a WdlObject or Namespace"))
+          } recoverWith {
+            case _ => Try(lookup(a.getAttribute("lhs").sourceString + "." + rhs.sourceString))
```

Comment:
Up until now, when the `TypeEvaluator` is created it was given a lookup function that works for everything except member accesses (which it gets in some convoluted way from the `from` parameter).

For WOM, we don't need the `from` and can just pass in a fully-stocked lookup (since we already know which lookups the evaluators are going to do). So we let the `from`-based lookup fail and fall back to the `lookup`-based lookup here.

If this survives the coming `scatter` shakeup then I think we'll probably want to parameterize how to do member access lookups rather than trying both and falling back if the first way fails